substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -1197,6 +1266,27 @@ private void computeFullFilePath() {
             }
         }
 
+        @Override
+        public boolean isConstructor() {
+            return method.isConstructor();
+        }
+
+        @Override
+        public boolean isVirtual() {
+            return method instanceof HostedMethod ? ((HostedMethod) method).hasVTableIndex() : false;
+        }
+
+        @Override
+        public boolean isOverride() {
+            GraalError.unimplemented("Use MethodInfo.isOverride() instead.");
+            return false;
```

Comment:
We'll need this one too.
```suggestion
            return method instanceof HostedMethod && allOverrides.contains(method);
```